analyzer/src/funTest/assets/projects/synthetic/gomod-dangling-embed/go.mod

```diff
@@ -0,0 +1,3 @@
+module gomod_dangling_embed
+
+go 1.19
```

Comment:
Shouldn't this also be 1.21.1?

Reply:
Why exactly should it be? (As-is it is consistent with the other test cases in `GoModFunTest`).

One aspect of the bug fix is in the tooling of version 1.21, but that tooling can be used with 1.19 projects, so I do not see an issue here.

Reply:
> One aspect of the bug fix is in the tooling of version 1.21, but that tooling can be used with 1.19 projects

That was the missing piece of information for me here.

Reply:
> That was the missing piece of information for me here.

Ok, so are we good to go?